Jenkinsfile

```diff
@@ -89,31 +93,33 @@ pipeline {
         // We want to avoid running in parallel.
         // When we have 2 build running on the same environment (gke env only) in parallel,
         // we get the error "gcloud crashed : database is locked"
-        stage ("Run Integration Tests on oss") {
-          steps {
-            script {
-              def tasks = [:]
-                tasks["Kubernetes GKE, oss"] = {
-                  sh "./bin/start --docker --oss --gke"
-                }
-              parallel tasks
-            }
-          }
-        }
+        // stage ("Run Integration Tests on oss") {
+        //   steps {
+        //     script {
+        //       def tasks = [:]
+        //         tasks["Kubernetes GKE, oss"] = {
+        //           sh "./bin/start --docker --oss --gke"
+        //         }
+        //       parallel tasks
+        //     }
+        //   }
+        // }
 
         stage ("Run Integration Tests on DAP") {
           steps {
             script {
               def tasks = [:]
-                tasks["Kubernetes GKE, DAP"] = {
-                  sh "./bin/start --docker --dap --gke"
-                }
-                tasks["Openshift v3.11, DAP"] = {
-                  sh "./bin/start --docker --dap --oc311"
-                }
-                tasks["Openshift v4.5, DAP"] = {
-                  sh "./bin/start --docker --dap --oc45"
+              tasks["Kubernetes GKE, DAP"] = {
+                sh "./bin/start --docker --dap --gke"
+              }
+              tasks["Openshift (Current), DAP"] = {
+                sh "./bin/start --docker --dap --current"
```

Comment:
thanks @BradleyBoutcher for getting this!
Can we verify that the OCP version is logged in a clear place so we know which versions are tested in each Jenkins run?